conda_forge_tick/mamba_solver.py

```diff
@@ -238,4 +238,7 @@ def _is_recipe_solvable_on_platform(recipe_dir, cbc_path, platform, arch):
         run_req = m.get_value('requirements/run', [])
         solvable &= mamba_solver.solve(run_req)
 
+        tst_req = m.get_values('test/requires', [])
```

Comment:
this might also include `requirements`, there are multiple `test` keys. Idk if that propagates to here though.

Reply:
Let me check on some recipes locally.

Reply:
I checked clangdev and the code above appears correct. tes/requires has only the test requirements in it, so it should be added to run.

Reply:
Ah ok, in the past we needed `requirements` as well so maybe that got fixed!

Reply:
Ohhhh shoot. Now I know what you mean. Yes we should add in test/requirements